src/Sentry.OpenTelemetry/DisabledSpanProcessor.cs

```diff
@@ -0,0 +1,24 @@
+using OpenTelemetry;
+
+namespace Sentry.OpenTelemetry;
+
+internal class DisabledSpanProcessor : BaseProcessor<Activity>
+{
+    private static readonly Lazy<DisabledSpanProcessor> LazyInstance = new();
+    internal static DisabledSpanProcessor Instance => LazyInstance.Value;
+
+    public DisabledSpanProcessor()
+    {
+        Console.WriteLine("Sentry is disabled so no OpenTelemetry spans will be sent to Sentry.");
```

Comment:
I think this should go through a diagnostic logger?

Reply:
> I think this should go through a diagnostic logger?

The DisabledSpanProcessor only gets used if Sentry is disabled, so none of the Sentry logging will work... Do you think there's a system diagnostic logging class that would be more useful than Console.WriteLine?